doc/nrf/libraries/others/audio_module.rst

```diff
@@ -0,0 +1,120 @@
+.. _lib_audio_module:
+
+Audio module
+############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The audio module library is an interface to audio processing functions that coordinate audio states and the exchange of audio-related data of an LE Audio application, such as :ref:`nrf53_audio_app`.
+
+Overview
+********
+
+The audio module is an interface for constructing custom audio processing modules, such as decoder, encoder, and I2S output.
+It provides a common interface to audio processing algorithms.
+
+Using this interface, you can open and configure the custom modules, connect to them, and start and stop them.
+You can also send audio data to and from the application.
+
+The operation of the module is determined by a set of functions that perform the processing.
+Connecting these modules lets you create the following audio system or stream:
+
+.. figure:: images/audio_module_stream.svg
```

Comment:
What is the difference between this figure and audio_module_example.svg?

Reply:
Nothing except that the audio module example illustrates an example system including the application.